src/database/gravity-db.c

```diff
@@ -155,15 +155,15 @@ bool gravityDB_open(void)
 	//            matches 'google.de' and all of its subdomains but
 	//            also other domains starting in google.de, like
 	//            abcgoogle.de
-	rc = sqlite3_prepare_v2(gravity_db,
+	rc = sqlite3_prepare_v3(gravity_db,
```

Comment:
> sqlite3_prepare_v3() has an extra "prepFlags" option that is used for special purposes.

Do we need the extra prepFlag? Are we having special purposes?

Reply:
Yes, see the extra new flag `SQLITE_PREPARE_PERSISTENT`. Explantation can be found in the commit message of https://github.com/pi-hole/FTL/pull/1255/commits/d134a4943ae7e937792f8ee1c6f9147d9a9cfa3a